src/config.py

```
@@ -2,168 +2,86 @@
+
 # =============================================================================
 # Omega Rule Configuration (Atlas paper Eq. 9)
 # =============================================================================
 
-# Context window for memory (c in the paper)
-# Only the last OMEGA_CONTEXT_WINDOW tokens contribute to memory
 OMEGA_CONTEXT_WINDOW: int = 256   # Context window size for Omega rule
-
-# Base decay rate for exponential weighting within context window
-# γ_base^(t-i) gives higher weight to recent tokens
 OMEGA_DECAY_BASE: float = 0.95    # Base decay rate for γ
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `γ` (GREEK SMALL LETTER GAMMA). Did you mean `y` (LATIN SMALL LETTER Y)?

(RUF003)